Packs/FiltersAndTransformers/Scripts/IfElif/README.md

```diff
@@ -0,0 +1,98 @@
+### A transformer for if-elif-else logic.
+
+The **If-Elif** transformer simulates a python *"if elif else"* tree using a JSON provided in the *conditions* argument.
+The JSON should be a list of dictionaries where all have the keys "*condition*", which holds a boolean expression, and "*return*", which holds the value to return if "*condition*" is evaluated to be true. To return a default value if all "*condition*"s were false, the last dictionary should have only the key "*default*" holding the valid JSON value. If this is not provided, an empty string will be returned as a default.
+In order to prevent injections, context values should be retrieved from the value entered in the *value (Get)* of the transformer with the hash-curly brackets `#{...}` syntax. This syntax has the same behavior as the classic XSOAR `${...}` syntax and uses the [Cortex XSOAR Transform Language (DT)](https://xsoar.pan.dev/docs/integrations/dt). To provide the full context to the transformer, use `${.}` as the *value (Get)* argument. **Note:** when used as a "*return*" value, this syntax should *not* be surrounded by quotes.  
+
+#### Supported operators for conditions:
+
+**Comparison operators** work like Python operators:
+| Operator | Name | Example |
+| --- | --- | --- |
+| == | Equal | x == y |
+| != | Not equal | x != y |
+| > | Greater than | x > y |
+| < | Less than | x < y |
+| >= | Greater than or equal to | x >= y |
+| <= | Less than or equal to | x <= y |
+| in | In | x in y|
+| not in | Not in | x not in y|
+
+**Logical operators** also follow the Python syntax:
+| Operator | Description | Example |
+| --- | --- | --- |
+| and | Returns True if both statements are true | x < 5 and x < 10 |
+| or | Returns True if one of the statements is true | x < 5 or x < 4 |
+| not | Reverse the result, returns False if the result is true | not(x < 5 and x < 10) |
+
+**Regular expressions** are implemented with the "regex_match" function, in the format: `regex_match('pattern', 'string')`. The behavior of the function is controlled with the *flags* argument.
+
+**Literal strings** should preferably be surrounded by single quotes. Do _not_ use `#{...}` in a string, instead, use the `+` operator. For example: `'first ' + #{second.string} + ' third'` will be equal to the common `"first ${second.string} third"`. (This method can be used for lists too.)
+
+The following flags can be used in the *flags**argument to control the transformer's behavior:
+|Flag|Effect|Example|
+|---|---|---|
+| case_insensitive | Comparisons between strings and regex matches are case-insensitive. | `'WoRd' == 'wOrD'` |
+| list_compare | Comparing an object with a list also compares the object with all values in the list and evaluates to true if *any* comparison is true.<br>Works on operators: `<` `>` `<=` `>=` `in` `not in` `+` | `'word' in ['word1', 'word2']` |
```

Comment:
please provide more examples. not clear to me why the example given needs list_compare to be true

Reply:
ben had this case where he needed to check if a string was contained in any string in a list. this would not work in python.